### PR TITLE
Add missing documentation for $ip variable

### PR DESCRIPTION
**Pull request by ypcs**
Filter `jpp_allow_login` actually accepts two variables (boolean and ip
address), and IP address is currently missing from the documentation.

As this variable is currently undocumented, current publicly available
documentation at <https://developer.jetpack.com/hooks/jpp_allow_login/>
is also misleading.

#### Changes proposed in this Pull Request:
This is small fix to documentation.

## Testing instructions:

* n/a

#### Does this pull request change what data or activity we track or use?
No changes to functionality.

### CODE REVIEW

File: projects/plugins/jetpack/modules/protect.php [hunk 1]
```diff
@@ -521,6 +521,7 @@ function is_current_ip_whitelisted() {
 		 * @since 4.4.0
 		 *
 		 * @param bool false Should we allow all logins for the current ip? Default: false
+		 * @param string ip IP address we're validating
```
Comment (jeherve) [inline projects/plugins/jetpack/modules/protect.php:524]:
```suggestion
		 * @param bool   $false Should we allow all logins for the current ip? Default: false.
		 * @param string $ip    IP address we are validating.
```

**Review by jeherve — DISMISSED** [review]
Thanks for the contribution!

Could you add a changelog entry for your change, as explained here:
https://github.com/Automattic/jetpack/blob/master/docs/monorepo.md#jetpack-changelogger

Thank you!

File: projects/plugins/jetpack/modules/protect.php (hunk 1 above)
Comment (ypcs) [reply projects/plugins/jetpack/modules/protect.php]: Thanks, fixed.

**Review by kraftbj — DISMISSED** [review]

**Review by kraftbj — APPROVED** [review]

**Comment by zinigor** [comment]
@ypcs sorry this got left for so long, we can't merge this anymore because it's not up to date with trunk. Would you like to rebase yourself? We can also apply your patch ourselves giving you credit for the commit.

**Comment by jeherve** [comment]
Since we've had issues with the Build action on this old PR, I've created #29731 to merge this change without any more issues, finally.

Thanks for your contribution @ypcs!

**Pull request closed without merging**